### ISSUE TITLE
[Bug]: only ice dragon eggs dont hatch

### ISSUE THREAD

**Issue by mrkayar** [Alex-the-666/Ice_and_Fire#5070]
### What happened?

in my modpack i found an ice dragon egg and tried the hatch but like 30 minutes nothing happended dragon egg turned red and i dont know i break the block and egg dissapeared i shared the mod names on log output

### Minecraft Version

1.20.1

### Ice and Fire version

iceandfire-2.1.13-1.20.21+build.beta.1.jar

### Citadel version

citadel-2.4.9-1.20.1.jar

### Relevant log output

```shell
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\ftb-library-forge-2001.1.4.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\drippyloadingscreen_forge_2.2.4_MC_1.20.1.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\fancymenu_forge_2.14.9-3_MC_1.20.1.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\konkrete_forge_1.6.1-2_MC_1.20.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\PackingTape-1.20.1-0.14.2.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\astikorcarts-redux-1.20.1-1.1.7.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\DistantHorizons-2.0.1-a-1.20.1.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\WeaponsOfMiracles-20.1.7.36.3.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\treeharvester-1.20.1-8.4.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\collective-1.20.1-7.28.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\playertrade-1.20.1-a.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\ShieldExpansion-1.20.1-1.1.7.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\torchmaster-20.1.5.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\smallships-forge-1.20.1-2.0.0-a2.3.3.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\Paraglider-forge-20.1.2.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\EpicStats1.0.7R fix 1.20.1.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\charmofundying-forge-6.4.5+1.20.1.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\charms-2.0.2-1.20.1.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\RingsOfAscension-1.20.1-2.0.1.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\elytraslot-forge-6.3.0+1.20.1.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\caelus-forge-3.1.0+1.20.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\Chunk Pregenerator-1.20-4.2.3.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\Terralith_1.20.4_v2.4.11.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\Butchersdelight Foods beta 1.20.1 1.0.2.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\Butchersdelight beta 1.20.1 2.0.8f.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\classicandsimplestatusbars-Forge-1.0.13.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\CreativeCore_FORGE_v2.11.16_mc1.20.1.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\ItemPhysic_FORGE_v1.7.0_mc1.20.1.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\another_furniture-forge-1.20.1-3.0.1.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\handcrafted-forge-1.20.1-3.0.5.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\resourcefullib-forge-1.20.1-2.1.20.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\artifacts-forge-9.2.0.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\curios-forge-5.4.7+1.20.1.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\cloth-config-11.1.118-forge.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\letsdo-nethervinery-forge-1.2.2.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\supplementaries-1.20-2.7.14.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\letsdo-beachparty-forge-1.1.3.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\waystones-forge-1.20-14.1.3.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\alexsmobs-1.22.7.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\moonlight-1.20-2.8.72-forge.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\pmmo-1.20.1-1.2.12.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\shadowizardlib-1.20.1-1.0.8.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\MedievalCraftWeapons-1.20.1-1.0.1.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\EpicFight-20.6.3.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\[1.20.1-forge]-Epic-Knights-8.11.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\dreadsteel-1.20.1-1.1.6.2.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\letsdo-candlelight-forge-1.2.7.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\letsdo-bakery-forge-1.1.3b.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\letsdo-meadow-forge-1.3.3c.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\letsdo-vinery-1.4.7-forge.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\doapi-1.2.8-forge.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\architectury-9.1.12-forge.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\mcw-doors-1.1.0forge-mc1.20.1.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\dummmmmmy-1.20-1.8.13.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\corpse-1.20.1-1.0.9.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\Aquaculture-1.20.1-2.5.1.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\citadel-2.4.9-1.20.1.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\cosmeticarmorreworked-1.20.1-v1a.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\BiomesOPlenty-1.20.1-18.0.0.592.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\TerraBlender-forge-1.20.1-3.0.0.169.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\sophisticatedbackpacks-1.20.1-3.19.4.978.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\create-1.20.1-0.5.1.f.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\sophisticatedcore-1.20.1-0.5.111.517.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\polymorph-forge-0.49.2+1.20.1.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\FarmersDelight-1.20.1-1.2.3.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\balm-forge-1.20.1-7.2.1.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\journeymap-1.20.1-5.9.18-forge.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\appleskin-forge-mc1.20.1-2.5.1.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\MouseTweaks-forge-mc1.20-2.25.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\jei-1.20.1-forge-15.2.0.27.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\iceandfire-2.1.13-1.20.1+build.beta-1.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\ftb-ranks-forge-2001.1.3.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\ftb-chunks-forge-2001.2.5.jar"
"C:\Users\aliba\curseforge\minecraft\Instances\KKK\mods\ftb-teams-forge-2001.1.4.jar"
```

### Other Information

- [X] I am using other mods besides Ice and Fire
- [X] I have attached a log file from the session that I have experienced this issue

**Comment by mrkayar** [comment]
and i didnt tried fire dragon yet bc i didnt found

**Comment by TheBv** [comment]
fixed with 03243f3f7529acbf1195a5de01d5c824c01c5d36
